<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-RQ8FSHG" failures="1" tests="12" name="test.LoginTest_Amazon" time="32.185" errors="0" timestamp="2023-04-30T14:59:11 EDT" skipped="0">
  <testcase classname="test.LoginTest_Amazon" name="validateNavigateToWishList" time="1.885"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validateNavigateToNewRelease" time="1.929"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validateCustomerLogin" time="1.976"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validatecustomerLogout" time="2.576"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="inValidEmail" time="0.490"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validEmailInValidPassword" time="0.742"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validateProductSearch" time="7.340"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validateSelectProduct" time="4.431"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validateselectDeliveryLocation" time="6.130">
    <failure message="Entered postal code is not as expected expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Entered postal code is not as expected expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at test.LoginTest_Amazon.validateselectDeliveryLocation(LoginTest_Amazon.java:131)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateselectDeliveryLocation -->
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validateSponsoredProduct" time="2.344"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validateAddToCart" time="1.320"/>
  <system-out/>
  <testcase classname="test.LoginTest_Amazon" name="validateCheckout" time="1.022"/>
  <system-out/>
</testsuite> <!-- test.LoginTest_Amazon -->
